test/functionalTest/cases/1316_create_subscription/post_subscriptions.test

```diff
@@ -74,6 +78,89 @@ echo
 echo
 
 
+echo "03. POST /v2/subscriptions"
```

Comment:
Maybe this should mirror the preamble, I mean:

```
echo "03. POST /v2/subscriptions (id)"
```

Similar comment for other steps.